quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/cli/PicocliTest.java

```diff
@@ -386,4 +386,29 @@ public void fastStartOptimizedSucceeds() {
         assertEquals(Integer.MAX_VALUE, nonRunningPicocli.exitCode); // "running" state
     }
 
+    @Test
+    public void wrongLevelForCategory() {
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("start-dev", "--log-level-org.keycloak=wrong");
+        assertEquals(CommandLine.ExitCode.USAGE, nonRunningPicocli.exitCode);
+        assertTrue(nonRunningPicocli.getErrString().contains("Invalid log level: wrong. Possible values are: warn, trace, debug, error, fatal, info."));
+    }
+
+    @Test
+    public void wildcardLevelForCategory() {
+        NonRunningPicocli nonRunningPicocli = pseudoLaunch("start-dev", "--log-level-org.keycloak=warn");
+        assertEquals(CommandLine.ExitCode.OK, nonRunningPicocli.exitCode);
+        var value = nonRunningPicocli.config.getConfigValue("quarkus.log.category.\"org.keycloak\".level");
+        assertEquals("quarkus.log.category.\"org.keycloak\".level", value.getName());
+        assertEquals("WARN", value.getValue());
+    }
```

Comment:
This is already tested [here](https://github.com/keycloak/keycloak/blob/d1f20193a7f1cda4c1da0bd15cf5a795e663ebb0/quarkus/runtime/src/test/java/org/keycloak/quarkus/runtime/configuration/test/ConfigurationTest.java#L536-L543).

Reply:
Same response as below.